### ISSUE TITLE
Sympa Accessibility Issue - 24 - X-UA-Compatible HTTP header must have the value IE=edge, was IE=100

### ISSUE THREAD

**Issue by rm1-uiuc** [sympa-community/sympa#1776]
The University of Illinois at Urbana-Champaign, Technology Services Accessibility Testing team conducted an Accessibility Review of Sympa on September 25, 2023 and found 26 issues that need to be fixed to make the interface accessible for visual impaired users.

Version
-------
6.2.72

Installation method
-------------------
Source Package

Expected behavior
-------------------
X-UA-Compatible HTTP header must have the value IE=edge, was IE=100 so content is rendered correctly to assure the Accessibility screen reader has no issues reading content.

Actual behavior
---------------
X-UA-Compatible HTTP header does not have a value IE=edge, IE=100

Steps to reproduce
------------------
Sympa Accessibility Issue - 24 - was determined by an Automation Tool Evaluation using AInspector and Siteimprove Accessibility Checkers as automated accessibility evaluation tools.

After running these automation tools the following issue was identified:

X-UA-Compatible HTTP header must have the value IE=edge, was IE=100.

HTML Code Validation

The purpose of code review is to make sure that the HTML follows the Document Type Declaration (DTD), which sets out the rules and grammar for the flavor of markup, thus enabling the browser to render the content accordingly. The HTML5 validator [NuValidator](https://validator.w3.org/nu/) was used in this evaluation. Code validation should be done to better conform with HTML standards and thus accessible coding practices. Although several of the errors uncovered by the Validator are minor, compliance means the site will operate as expected when used by assistive technology (AT) (WCAG 2.0 [4.1.1 Parsing](https://www.w3.org/TR/UNDERSTANDING-WCAG20/ensure-compat-parses.html)). Often, fixing one or two of those errors early on can resolve many of the errors flagged, since it has a waterfall effect.

Additional information
----------------------
Attached is the Accessibility Review conducted on Sept 25,2023
[Tech Services Review - Accessibility Evaluation Report for Sympa.pdf](https://github.com/sympa-community/sympa/files/13562607/Tech.Services.Review.-.Accessibility.Evaluation.Report.for.Sympa.pdf)

**Comment by ikedas** [comment]
Nonstandard `X-UA-Compatible` HTTP response header field has been used for the obsoleted Internet Explorer and it may be remoed now.  PR #1679 will make such a change and will close this issue.
